### PR TITLE
fix: Profile appear as anonymous when decoding asset key fails

### PR DESCRIPTION
**Pull request by dzbo**
### Ticket ID

[DEV-9566](https://app.clickup.com/t/86bx18mt2)

### Description

- split fetching keys (LSP3, LSP5, LSP12) so when one fails it won't affect the others

### CODE REVIEW

**Comment by Hugoo** [comment]
Task linked: [DEV-9566 Profile appear as anonymous when decoding asset key fails](https://app.clickup.com/t/2645698/DEV-9566)

File: utils/fetchAssets.ts [hunk 1]
```diff
@@ -1,18 +1,52 @@
...
-      (await assetRepo.loadAssets(issuedAssetAddresses, profileAddress))
+    try {
+      receivedAssets = (await erc725.fetchData('LSP5ReceivedAssets[]'))?.value
+      assertArray(receivedAssets)
+      assertAddresses(receivedAssets)
+
+      receivedAssets &&
+        (await assetRepo.loadAssets(receivedAssets, profileAddress))
+    } catch (error) {
+      throw error
```
Comment (richtera) [inline utils/fetchAssets.ts:32]: The separation makes sense but throwing an error would still not do the next one? Seems problematic

**Review by richtera — CHANGES_REQUESTED** [review]
Seems problematic to throw the error

File: utils/fetchAssets.ts (hunk 1 above)
Comment (dzbo) [reply utils/fetchAssets.ts:32]: Right, I'll fix this. I mainly focused on separation between profile and assets, not between issued/received assets. But that will also be useful.

**Review by richtera — APPROVED** [review]

**Pull request merged**